lib/util/mime_utils.js

```diff
@@ -96,6 +96,55 @@ shaka.util.MimeUtils = class {
     return codecs.split(',');
   }
 
+  /**
+   * Get the normalized codec from a codec string,
+   * ignoring multiple names of a same codec.
+   *
+   * @param {string} codecString
+   * @return {string}
+   */
+  static getNormalizedCodec(codecString) {
+    const parts =
+      shaka.util.MimeUtils.getCodecParts_(codecString.toLowerCase());
+    const base = parts[0];
+    const profile = parts[1];
+    switch (true) {
+      case base === 'mp4a' && profile === '66':
+      case base === 'mp4a' && profile === '67':
+      case base === 'mp4a' && profile === '68':
+        return 'mpeg2_aac';
+      case base === 'mp4a' && profile === '69':
+      case base === 'mp4a' && profile === '6b':
+        return 'mp3';
+      case base === 'mp4a' && profile === '40.2':
+      case base === 'mp4a' && profile === '40.02':
+      case base === 'mp4a' && profile === '40.5':
+      case base === 'mp4a' && profile === '40.05':
+      case base === 'mp4a' && profile === '40.29':
+        return 'mpeg4_aac';
+      case base === 'mp4a' && profile === '40.42':
+        return 'mpeg4_xhe_aac'; // Extended HE-AAC
+      case base === 'mp4a' && profile === 'a5':
+        return 'ac-3'; // Dolby Digital
```

Comment:
I have doubts here, because this would imply that we can adapt between mp4/ac-3 and mp4/mp4a.a5 and I don't know if this is viable or not...

Reply:
Same for ec-3, dtsx and dtsc

Reply:
I removed these codecstring since we don't know if shaka would play them together. We can always add them later if we ever encounter mp4a.?? ac-3/ec-3 and find that browsers can transition from one to another.

Reply:
I keep seeing this, maybe you forgot to push?

Reply:
oh sorry :(